code/modules/client/preference_setup/loadout/items/xeno/human.dm

```diff
@@ -19,6 +19,7 @@
 	whitelisted = list(SPECIES_HUMAN_OFFWORLD)
 	sort_category = "Xenowear - Human"
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+	allowed_roles = list("Chief Engineer", "Engineer", "Atmospheric Technician", "Engineering Apprentice", "Machinist", "Engineering Personnel")
```

Comment:
Unwanted commit. Claims to just be because it provides welding protection. If that's the case, then I'd much rather see the protection removed than by role restricting an item like this.